src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -6,6 +6,7 @@ public net.minecraft.block.Block$Properties func_200947_a(Lnet/minecraft/block/S
 public net.minecraft.block.Block$Properties func_200951_a(I)Lnet/minecraft/block/Block$Properties; # lightValue
 public net.minecraft.block.Block$Properties func_208770_d()Lnet/minecraft/block/Block$Properties; # variableOpacity
 public net.minecraft.block.Block$Properties func_222380_e()Lnet/minecraft/block/Block$Properties; # noDrops
+public net.minecraft.block.TNTBlock func_196535_a(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/entity/LivingEntity;)V # explode
```

Comment:
Keep this private, we dont want to make a function public and then ask nicely for people not to use it.
Make TNTBlock override explode and call the private method.

However, overall this PR needs to not have 'is explosive/explode' it just needs 'catchFire' and let the block manage what happens itself. Why does the outside caller care if it's explosive?

Reply:
I will update it, but how should I then handle the IDispenseItemBehavior patch. Should I check `isFlammable` then calculate based on the source what face we are asking about or do I need to add some other check

Reply:
Make catchFire return a boolean?

Reply:
Sounds good, and I think the information can be figured out, I will look at it tomorrow.